tests/components/srp_energy/test_init.py

```diff
@@ -1,28 +1,20 @@
 """Tests for Srp Energy component Init."""
-from homeassistant import config_entries
-from homeassistant.components import srp_energy
-from homeassistant.core import HomeAssistant
+from homeassistant.components.srp_energy import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
 
-from . import init_integration
 
+async def test_setup_entry(hass, init_integration):
+    """Test setup entry."""
+    assert init_integration.state == ConfigEntryState.LOADED
+    assert hass.data[DOMAIN][init_integration.entry_id]
```

Comment:
Please don't assert things in hass.data. That's integration details.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations